src/main/groovy/com/bmuschko/gradle/docker/internal/DefaultDockerConfigResolver.groovy

```diff
@@ -0,0 +1,66 @@
+package com.bmuschko.gradle.docker.internal
+
+import com.google.common.annotations.VisibleForTesting
```

Comment:
Let's not explicitly use a Guava class. It may cause issues down the road as versions are often times breaking + this is just a transitive dependency of something else. We didn't declare Guava to be use in the plugin.

Reply:
Done.